src/vcpkg-fuzz/main.cpp

```diff
@@ -16,18 +16,6 @@ using namespace vcpkg;
 
 namespace
 {
-    DECLARE_AND_REGISTER_MESSAGE(FuzzInvalidKind, (msg::value), "example of {value} is 'utf-8'", "invalid kind: {}");
```

Comment:
Hmmm I think the right thing for these is actually to unlocalize them. The fuzz tester is just for our team, not something we ship.

Reply:
Yes, it's causing messages to show up in messages.json which aren't actually used in vcpkg. (So translators will waste time translating them, etc.)

This didn't matter in the old design because the DECLARE_AND_REGISTER_MESSAGEs didn't put anything in a place vcpkg.exe's generate-message-map command would see.

Reply:
Gotcha, I've reverted main.cpp to its original state. I will unlocalize the messages and remove the DECLARE_AND_REGISTER_MESSAGES in a separate PR.